### DISCUSSION TITLE
Destination path

### DISCUSSION THREAD

**Discussion by TinyEmerald** [ImageMagick/ImageMagick#5431] [Help] [answered]
Hi
Does anyone know how to create a new image in a specific folder using a command line in Windows (10)? Creating the file is no problem, but I can't figure out how to specify the destination path. I tried -path but that doesn't work. Thanks.

magick -size 320x85 canvas:none -font Arial -pointsize 23 -draw "text 25,60 \'My \n batch\'" -channel RGBA -blur 0x6 -fill white -stroke white  -draw "text 20,55 \'My \n batch\'" -path c:\ fuzzy-magick.png

**Comment by snibgo** [comment, marked as answer]
"`-path`" is only for "`magick mogrify`", because that command takes an input filename, with no explicit filename.

The simpler "`magick`" command takes an explicit output filename, and you can specify the folder there. The output filename is usually the final argument. Like this:
```
magick -size 320x85 canvas:none -font Arial -pointsize 23 -draw "text 25,60 'My \n batch'" -channel RGBA -blur 0x6 -fill white -stroke white -draw "text 20,55 'My \n batch'" c:\fuzzy-magick.png
```

**Reply by TinyEmerald** [reply]
Thank you! I tried it and it works if I run Command Prompt as an administrator.

**Comment by snibgo** [comment]
You shouldn't need administrator privileges. Perhaps you installed IM administrator, and didn't install it for other users.

Some people run everything as administrator. I don't, because that would allow hostile or faulty programs to do bad things.
